python/aibrix_kvcache/aibrix_kvcache/profiling.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2024 The Aibrix Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import functools
+import hashlib
+from contextlib import contextmanager
+
+from . import envs
+
+_NVTX_COLORS = ["green", "blue", "yellow", "purple", "rapids", "red"]
+
+
+@functools.lru_cache
+def _nvtx_get_color(name: str):
```

Comment:
seems the primary logic is similar as previous version, just the code refactor (common -> profling). lgtm